README.md

```diff
@@ -114,6 +114,26 @@ TEST_HEADLESS=false yarn test:cucumber
 
 Note: By default tests are run heedlessly (i.e. without displaying the browser) setting the `TEST_HEADLESS` flag to `false` will open the test browser window allowing you to see whats happening in realtime.
 
+Running a single test:
+
+```bash
+yarn test:cucumber:grep 'Name of Feature, Scenario, or @tag'
+```
+
+Viewing functional test reports locally:
+
+```bash
+yarn test:cucumber:reports
+```
+
+To view the report from Jenkins, navigate to the build artefacts page scroll to the bottom and click “Download All”.
+
+Then extract it and run:
+
+```bash
+npx allure-commandline serve ~/Downloads/archive/functional-output/functional/reports
```

Comment:
maybe add a curl or wget command to download the report?

Reply:
That would be pretty nifty if we could do that. Though might be a tricky getting the right cookies and getting past the portal endpoint inspector